plugin/evm/block_builder.go

```diff
@@ -169,11 +169,17 @@ func (b *blockBuilder) awaitSubmittedTxs() {
 					// Give time for this node to build a block before attempting to
 					// gossip
 					time.Sleep(waitBlockTime)
-					// [GossipTxs] will block unless [gossiper.txsToGossipChan] (an
-					// unbuffered channel) is listened on
-					if err := b.gossiper.GossipTxs(ethTxsEvent.Txs); err != nil {
+
+					// Retrieve nodes to gossip to
+					proposers, err := b.ctx.ProposerRetriever.GetCurrentProposers(context.TODO()) // UNSURE OF WHAT CONTEXT TO USE HERE
+					if err != nil {
+						log.Warn("failed to retrieve list of proposers", "err", err)
+					}
```

Comment:
let's be careful here to not call `GossipTxsToNodes` with `proposers = nil` or guard against it explicitly in `GossipTxsToNodes`.

Reply:
Opted for guarding against it in `GossipTxsToNodes`